pkg/providers/nutanix/config/cp-template.yaml

```diff
@@ -0,0 +1,196 @@
+piVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: NutanixCluster
+metadata:
+  name: "{{.clusterName}}"
+  namespace: "{{.eksaSystemNamespace}}"
+spec:
+  prismCentral:
+    address: "{{.nutanixEndpoint}}"
+    port: {{.nutanixPort}}
+    insecure: true
```

Comment:
Can we make this configurable instead of hardcoding it to true?

Reply:
good catch; for sure. We want this configurable only for the case where user doesn't have publicly trusted CA certs.